From 3db14376ae8d58c369638610795a81029d5db953 Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Thu, 24 Aug 2023 12:55:38 +0800
Subject: [PATCH] 0002-remove-some-broken-tests-in-deepin

---
 test/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/test/Makefile b/test/Makefile
index 74c40b3..f08cede 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -4,7 +4,6 @@ OUR_TESTS= \
 	annotate-output \
 	checkbashisms \
 	dd-list \
-	debchange \
 	debdiff \
 	debrepro \
 	debsign \
@@ -16,7 +15,6 @@ OUR_TESTS= \
 	uscan_ftp \
 	uscan_git \
 	uscan_svn \
-	uscan_mangle \
 	uscan_group \
 	perltidy \
 	wrap-and-sort \
-- 
2.33.1

